packages/gravity-ui-web/src/sass/01-tools/_layouts.scss

```diff
@@ -0,0 +1,53 @@
+// a mixin to add column based layout styles
+@mixin grav-l-column-layout($column-count: 2, $column-spacing: $grav-sp-m) {
+  display: flex;
+  flex-wrap: wrap;
+  padding-right: 0;
+  padding-left: 0;
+
+  > * {
```

Comment:
Having seen what you do later on for `.grav-c-list-cards-basic`, I have a proposal: Why not change this mixin to work the same way that does (and thus save yourself a few overrides there later on)? I.e.:

* Set negative left & right half-gutter margins on the container
* Remove the `nth-child()` rules below that zero the outer margins of the left and right-most items

I tried it quickly locally and it appears to work fine.

This has the added bonus that we could make a generic version of `.grav-c-list-cards-basic__item--callout` that can stretch a single item of a "no-danglies" column layout to be full-width. (As it stands, that would throw the `nth-child` logic out of whack since all subsequent items would be off by one)